C/c4Document.cc

```diff
@@ -596,7 +638,17 @@ C4Document* c4doc_update(C4Document *doc,
 
 bool c4doc_removeRevisionBody(C4Document* doc) noexcept {
     auto idoc = asInternal(doc);
-    return idoc->mustBeInTransaction(NULL) && idoc->removeSelectedRevBody();
+    if(!idoc->mustBeInTransaction(nullptr)) {
+        WarnError("c4doc_removeRevisionBody: Called outside transaction");
```

Comment:
Wouldn't it be better to put this warning in `mustBeInTransaction`?

Reply:
Maybe, but it wouldn't indicate which function called it incorrectly.